src/Emulator/Tabs/ConfigureTab/DeviceContainer/DeviceContainer.tsx

```diff
@@ -85,6 +85,19 @@ const DeviceContainer = React.forwardRef<HTMLDivElement, Props>(
   ({ deviceName, devices, addDevice }, ref) => {
     const { dispatch } = useEmulator();
 
+    useEffect(() => {
+      if (DeviceContainerContentRef.current){
+        // Convert to HTMLDivElement, because TypeScript complains that
+        // .scrollTop and .scrollHeight does not exist.
+        // Convert to unknown first, because TypeScript complains that
+        // 'null' and 'HTMLDivElement' do not sufficiently overlap.
+        const element = DeviceContainerContentRef.current as unknown as HTMLDivElement;
+        element.scrollTop = element.scrollHeight;
+      }
+    }, [devices]);
+
+    const DeviceContainerContentRef = useRef(null);
```

Comment:
You can add an `HTMLDivElement` as a generic for the `useRef` instead of casting to unkown

```suggestion
        const element = DeviceContainerContentRef.current;
        element.scrollTop = element.scrollHeight;
      }
    }, [devices]);

    const DeviceContainerContentRef = useRef< HTMLDivElement>(null);
```